docs/coordinates/velocities.rst

```diff
@@ -436,8 +437,9 @@ the same: the radial velocity should be essentially the same in both frames:
                radial_velocity=[10]*1000*u.km/u.s,
                location=location, obstime=time)
     gcrs = aa.transform_to(GCRS(obstime=time))
-    plt.plot_date(time.plot_date, gcrs.radial_velocity.to(u.km/u.s))
-    plt.ylabel('RV [km/s]')
+    fig, ax = plt.subplots()
+    ax.plot_date(time.plot_date, gcrs.radial_velocity.to(u.km / u.s))
+    ax.set_ylabel("RV [km/s]")
```

Comment:
The code here is drawing two plots when only one is expected.

Reply:
nice catch